yieldscan.config.js

```diff
@@ -51,11 +65,18 @@ export const networks = [
 		network: "westend",
 		isTestNetwork: true,
 		denom: "WND",
-		feesAddress: process.env.NEXT_PUBLIC_WESTEND_FEES_ADDRESS,
+		feesAddress: isNil(process?.env?.NEXT_PUBLIC_WESTEND_FEES_ADDRESS)
+			? false
+			: process.env.NEXT_PUBLIC_WESTEND_FEES_ADDRESS,
 		feesEnabled: isNil(process?.env?.NEXT_PUBLIC_WESTEND_FEES_ENABLED)
 			? false
 			: JSON.parse(process.env.NEXT_PUBLIC_WESTEND_FEES_ENABLED),
-		feesRatio: process.env.NEXT_PUBLIC_WESTEND_FEES_RATIO,
+		feesRatio: isNil(process?.env?.NEXT_PUBLIC_WESTEND_FEES_RATIO)
+			? 0
+			: JSON.parse(process.env.NEXT_PUBLIC_WESTEND_FEES_RATIO),
+		lastDiscountDate: isNil(process?.env?.NEXT_PUBLIC_LAST_DISCOUNT_DATE)
```

Comment:
@sahilnanda1995 I have changed the .env file. The lastDiscountDate is now stored in the .env file. Please store 

NEXT_PUBLIC_LAST_DISCOUNT_DATE = "31 Dec 2021 23:59:59 UTC"

in the .env file and then do the transaction again. It would prolly work correctly.

Reply:
okay